docs/contributor/03-10-release.md

```diff
@@ -21,14 +21,19 @@ To create a release, follow these steps:
    iii. click  **Run workflow** on the right  
    iv. provide a version, for example, 1.2.0  
    v. choose real or dummy credentials for Service Manager  
-2. The GitHub action, defined in the [`create-release.yaml`](/.github/workflows/create-release.yaml) file, creates a GitHub tag and draft release with the provided name.
-3. The GitHub action asynchronously initiates unit tests and E2E tests jobs. E2E upgrade tests run only with real credentials for Service Manager.
-4. The tag creation triggers Prow Jobs, `post-btp-manager-module-build` and `post-btp-manager-build`, defined in [btp-manager-build.yaml](https://github.com/kyma-project/test-infra/blob/main/prow/jobs/btp-manager/btp-manager-build.yaml).
-5. `post-btp-manager-build` builds a Docker image tagged with the release name.
-6. `post-btp-manager-module-build` runs the `kyma alpha create module` command, which creates a Kyma module and pushes the image to the registry. Kyma CLI is called with the `--sec-scanners-config` flag and uses a dynamically created file to configure security scanning settings in the module template. Finally, the job uploads the `template.yaml`,`template_control_plane.yaml`, `btp-manager.yaml` and `btp-operator-default-cr.yaml` files to the btp-manager release as release assets.
-7. The GitHub action waits for the `template.yaml` asset in the GitHub release and for images in the Docker registry.
-8. The GitHub action fetches the module image and runs E2E tests on the k3s cluster with the specified credentials. 
-9. If the unit tests and E2E tests are completed successfully, the GitHub action publishes the release.
+   vi. choose to bump or not the security scanner config
```

Comment:
```suggestion
   vi. choose whether to bump or not to bump the security scanner config file
```

Reply:
@IwonaLanger suggested the same change, but without the last word `file`. I think without the word it better describes what is happening. We are not bumping the file, we are bumping the "config". Or it's just me misunderstanding something.